src/components/CoursePage/ReviewContainter.js

```diff
@@ -0,0 +1,129 @@
+import { React, useRef, useState } from "react";
+import style from "../../style_modules/CoursePage/ReviewContainter.module.css";
+import ReviewOfUser from "./ReviewOfUser";
+/**
+ *
+ * @param {object} courseInfo conatain all data related with course
+ * @returns
+ */
+function ReviewContainter({ courseInfo }) {
+  const [show, setShow] = useState(false);
+  const [rate, setRate] = useState(-1);
+  const [render, setRender] = useState(0);
+  const inputref = useRef("");
+  const [Search, setSearch] = useState([]);
+  const filter = () => {
+    let str = inputref.current.value;
+    console.log(str);
+    if (str !== "") {
+      setSearch(str.split(" "));
+    } else setSearch([]);
+  };
+  const Data_Reviews = [];
+  const Words_Bold = [];
+  courseInfo.map((item) => {
+    let res = item.review.split(" ");
```

Comment:
add comments to res and ok variable what is the responsibility of these variables.